editor/osm_auth.cpp

```diff
@@ -113,12 +136,12 @@ OsmOAuth::SessionID OsmOAuth::FetchSessionId(string const & subUrl, string const
   if (!request.RunHttpRequest())
     MYTHROW(NetworkError, ("FetchSessionId Network error while connecting to", url));
   if (request.WasRedirected())
-    MYTHROW(UnexpectedRedirect, ("Redirected to", request.UrlReceived(), "from", url));
+    MYTHROW(UnexpectedRedirect, ("FetchSessionId Unexpected redirected to", request.UrlReceived(), "from", url));
   if (request.ErrorCode() != HTTP::OK)
     MYTHROW(FetchSessionIdError, (DebugPrint(request)));
 
   SessionID sid = { request.CombinedCookies(), FindAuthenticityToken(request.ServerResponse()) };
-  if (sid.m_cookies.empty() || sid.m_token.empty())
+  if (sid.m_cookies.empty() || sid.m_authenticity_token.empty())
```

Comment:
m_authenticityToken

Reply:
Renamed`m_authenticity_token` to `m_authenticityToken`